node/bootstrapper.go

```diff
@@ -43,16 +43,20 @@ func (*Bootstrapper) Bootstrap(c map[string]interface{}) error {
 func getServers(ctx map[string]interface{}) ([]Server, error) {
 	p2pSrv, ok := ctx[bootstrap.BootstrappedP2PServer]
 	if !ok {
-		return nil, fmt.Errorf("p2p server not initialised")
+		return nil, fmt.Errorf("p2p server not initialized")
 	}
 
 	apiSrv, ok := ctx[bootstrap.BootstrappedAPIServer]
 	if !ok {
-		return nil, fmt.Errorf("API server not initiliase")
+		return nil, fmt.Errorf("API server not initialized")
+	}
+
+	queueSrv, ok := ctx[bootstrap.BootstrappedQueueServer]
```

Comment:
```suggestion
	queueSrv, ok := ctx[bootstrap.BootstrappedQueueServer].(Server)
```

Can we change this to above two servers as well?

Reply:
If the BootstrappedQueueServer is present with a type that doesn't implement server, then this will panic